app/static/src/scss/includes/_static-content-template.scss

```diff
@@ -1,9 +1,15 @@
-/* SCSS for How-to-use-this-service page */
+/* SCSS for static content template */
 
 .govuk-grid-column-one-third {
   &--ayr-content {
     position: sticky;
+    max-height: 100dvh;
```

Comment:
```suggestion
    max-height: calc(100dvh - 1rem);
    overflow: auto;
```

Reply:
The box is 1rem from the top so we have to subtract that from the height.

I've then added an overflow so that if it is shorter then you get scrollbars.

Reply:
I can't review this because I don't have a local dev env set up for AYR 😅